pkg/graveler/ref/manager_test.go

```diff
@@ -161,6 +186,9 @@ func TestManager_DeleteRepository(t *testing.T) {
 			t.Fatalf("unexpected error: %v", err)
 		}
 
+		// wait for cache expiry
+		time.Sleep(ref.RepositoryCacheExpiry + ref.RepositoryCacheJitter + time.Second)
```

Comment:
So long....

Reply:
Could be avoided if provided a unique repo name

Reply:
Can't be avoided - this is the test - delete + verify + create again.

Reply:
We can have the cache configuration injected and configure it to 0 for tests.